homeassistant/components/huum/light.py

```diff
@@ -0,0 +1,61 @@
+"""Control for light."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from homeassistant.components.light import ColorMode, LightEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from .coordinator import HuumConfigEntry, HuumDataUpdateCoordinator
+from .entity import HuumBaseEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: HuumConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up light if applicable."""
+    coordinator = config_entry.runtime_data
+
+    # Light is configured for this sauna.
+    if coordinator.data.config >= 2:
+        async_add_entities([HuumLight(coordinator)], True)
+
```

Comment:
```suggestion
        async_add_entities([HuumLight(coordinator)])

```
we don't need to update coordinator entities